jobs/cloud_controller_ng/templates/bpm.yml.erb

```diff
@@ -20,7 +20,9 @@ cloud_controller_ng_config = {
     "BUNDLE_GEMFILE" => "/var/vcap/packages/cloud_controller_ng/cloud_controller_ng/Gemfile",
     "CLOUD_CONTROLLER_NG_CONFIG" => "/var/vcap/jobs/cloud_controller_ng/config/cloud_controller_ng.yml",
     "C_INCLUDE_PATH" => "/var/vcap/packages/libpq/include",
+<% if p("export_dyno") %>
```

Comment:
It looks like this is not the only place we set `DYNO` - can you please make sure to have this property control all places where that is being set?